### DISCUSSION TITLE
Designing in Browser

### DISCUSSION THREAD

**Discussion by timgavin** [hydephp/hyde#259] [Q&A] [answered]
Is it possible to design my site in the browser using Hyde? Currently it looks like I have to run two commands in the terminal:

**terminal window 1**
```
php hyde serve
```

**terminal window 2**
```
npm run watch
```

and then manually refresh the browser after every design change. Or is there a way to get the browser to refresh automatically after every design change, like with Vite? I looked through the docs but didn't see anything to address this; did I miss something?

**Comment by emmadesilva** [comment, marked as answer]
Hi! Hyde no longer ships with BrowserSync due to its constant security vulnerabilities. However, it should be fairly easy for you to add it back to your project if you take a look at the diff in the removing PR https://github.com/hydephp/hyde/pull/93/files

I have considered trying to add a livereload feature to the realtime compiler, seeing as we now have a feature request for this, I can increase the priority of that!

**Reply by emmadesilva** [reply]
@timgavin Just wanted to let you know that HydePHP v2.0 will ship with Vite! And bring live reload and HMR.

Also, in the meantime, if you just want to get live Tailwind reloads, you can enable the "Play CDN" feature in the Hyde config, that will render the Tailwind styles in the browser which is great for development as you don't need to rerun Mix until you're ready to compile for production.
